Gemfile

```diff
@@ -59,9 +59,11 @@ gem 'rails_12factor', group: :production
 gem 'mocha', '~> 1.1', group: :test
 gem "spring", group: :test
 gem 'autoprefixer-rails', '~> 8.4'
+gem 'json-schema', '~> 2.8'
```

Comment:
Why use a JSON schema validator when there are YAML validators available? For example: [kwalify](http://www.kuwata-lab.com/kwalify/).

Reply:
Oh. I didn't see this. http://www.kuwata-lab.com/kwalify/ looks interesting

Reply:
I used https://www.jsonschema.net/ to generate JSON schema but there's nothing similar for YAML. To save time writing schema(that's hundreds of lines), I think JSON schema validator would be a better choice for now.